connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorHelper.java

```diff
@@ -54,14 +54,14 @@ public static Map<String, Object> createOutputVariables(
       outputVariables.put(resultVariableName, responseContent);
     }
 
-    Optional.ofNullable(resultExpression)
-        .filter(s -> !s.isBlank())
-        .map(
-            expression ->
-                FEEL_ENGINE_WRAPPER.evaluateToJson(
-                    expression, responseContent, wrapResponse(responseContent)))
-        .map(json -> parseJsonVarsAsTypeOrThrow(json, Map.class, resultExpression))
-        .ifPresent(outputVariables::putAll);
+    if (resultExpression != null && !resultExpression.isBlank()) {
```

Comment:
```suggestion
    if (StringUtils.isNotBlank(resultExpression)) {
```

Reply:
@johnBgood What is the benefit over: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#isBlank() ?

Reply:
the `null` check, and conciseness I guess